build.gradle

```diff
@@ -41,40 +45,48 @@ dependencyManagement {
 }
 
 dependencies {
+    // Declared
+    compile 'org.codehaus.groovy:groovy-json'
+    compile "org.hibernate:hibernate-ehcache:5.4.0.Final"
+
+    // Grails Generated
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
     compile "org.springframework.boot:spring-boot-starter-logging"
     compile "org.springframework.boot:spring-boot-autoconfigure"
-    compile "org.grails:grails-core"
-    compile "org.springframework.boot:spring-boot-starter-actuator"
     compile "org.springframework.boot:spring-boot-starter-tomcat"
-    compile "org.grails:grails-dependencies"
-    compile "org.grails:grails-web-boot"
-    compile "org.grails.plugins:cache"
-    compile "org.grails.plugins:scaffolding"
+    compile "org.grails:grails-core"
+    compile "org.grails:grails-plugin-i18n"
+    compile "org.grails:grails-plugin-url-mappings"
```

Comment:
Is this needed?

Reply:
If I remove this one, I get the following during ```check```:

```text
ERROR grails.plugins.DefaultGrailsPluginManager - ERROR: Plugin [controllers] cannot be loaded because its dependencies [['core', 'i18n', 'urlMappings']] cannot be resolved
ERROR grails.plugins.DefaultGrailsPluginManager - ERROR: Plugin [converters] cannot be loaded because its dependencies [['controllers', 'domainClass']] cannot be resolved
```

However, the test does pass.